packages/examples/fortune-v2/reputation-oracle/server/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@human-protocol/reputation-oracle",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
+  "scripts": {
+    "build": "nest build",
+    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
+    "start:prod": "node dist/main",
+    "migration:create": "typeorm-ts-node-commonjs migration:create",
+    "migration:generate": "typeorm-ts-node-commonjs migration:generate -d typeorm.config.ts",
+    "migration:revert": "typeorm-ts-node-commonjs migration:revert -d typeorm.config.ts",
+    "migration:run": "typeorm-ts-node-commonjs migration:run -d typeorm.config.ts",
+    "migration:show": "typeorm-ts-node-commonjs migration:show -d typeorm.config.ts",
+    "docker:db:up": "docker-compose up -d && yarn migration:run",
+    "docker:db:down": "docker-compose down",
```

Comment:
```suggestion
    "docker:db:up": "docker compose up -d && yarn migration:run",
    "docker:db:down": "docker compose down",
```

Reply:
https://docs.docker.com/compose/migrate/ in my machine no difference what kind of signature to use, lets update it